excercsises/2_page_object/pages/home.js

```diff
@@ -9,13 +9,20 @@ var HomePage = function() {
   this.getTitle = function() {
     return browser.driver.getTitle();
   };
-  this.getItemsFromContactPage = ()=>{
-    return element(by.xpath('//*/span/h3')),element(by.xpath('//*/fieldset/h3'));
-  }
-  this.getContactText = () => {
+  this.findAdressAndgetInTouch=  async ()=>{
```

Comment:
I think, the method name is still a little wrong. Try to think what the method is doing. You know that it is returning two strings of the titles on the page, the word 'get' was the good part. I would name it something like 'getHeaders' as both 'Get in touch' and 'Address' are marked as h3 on the page or 'getSectionTitles'